File: problemset/theatre-square/theatre-square.c
#include <stdio.h>
#include <math.h>

int main()
{
    double m, n, a;
    scanf("%lf", &m);
    scanf("%lf", &n);
    scanf("%lf", &a);
    if (m >=1 & m <= pow(10, 9) & n >=1 & n <= pow(10, 9) & a >=1 & a <= pow(10, 9))
    {
        long long int result = ceil((m / a)) * ceil((n / a));
        printf("%lld", result);
    }

}

File: problemset/theatre-square/theatre-square.py
from math import ceil
text = input()
remove_spaces = text.split(" ")
remove_spaces = [int(i) for i in remove_spaces]

if (1 <= remove_spaces[0] <= 10**9 and 1 <= remove_spaces[1] <= 10**9 and 1 <= remove_spaces[2] <= 10**9):
    r = ceil(remove_spaces[0] / remove_spaces[2]) * ceil(remove_spaces[1] / remove_spaces[2])

print(r)